frontend/src/metabase/query_builder/selectors.js

```diff
@@ -1052,3 +1053,25 @@ export function getEmbeddedParameterVisibility(state, slug) {
   const embeddingParams = card.embedding_params ?? {};
   return embeddingParams[slug] ?? "disabled";
 }
+
+export const getSubmittableQuestion = (state, question) => {
+  const series = getTransformedSeries(state);
+  const resultsMetadata = getResultsMetadata(state);
```

Comment:
This hasn't been 100% moved from `core.js`: https://github.com/metabase/metabase/blob/244ad7e/frontend/src/metabase/query_builder/actions/core/core.js#L238-L246

I'll move it as a part of #37009